grammars/css.cson

```diff
@@ -749,30 +1823,121 @@
     'beginCaptures':
       '0':
         'name': 'punctuation.definition.string.begin.css'
-    'end': '"'
+    'end': '"|(?<!\\\\)(?=$|\\n)'
     'endCaptures':
       '0':
         'name': 'punctuation.definition.string.end.css'
     'name': 'string.quoted.double.css'
     'patterns': [
       {
-        'match': '\\\\(\\h{1,6}|.)'
-        'name': 'constant.character.escape.css'
+        'begin': '(?:\\G|^)(?=(?:[^\\\\"]|\\\\.)+$)'
+        'end': '$'
+        'name': 'invalid.illegal.unclosed.string.css'
+        'patterns': [
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        'include': '#escapes'
       }
     ]
   'string-single':
     'begin': '\''
     'beginCaptures':
       '0':
         'name': 'punctuation.definition.string.begin.css'
-    'end': '\''
+    'end': "'|(?<!\\\\)(?=$|\\n)"
     'endCaptures':
       '0':
         'name': 'punctuation.definition.string.end.css'
     'name': 'string.quoted.single.css'
     'patterns': [
       {
-        'match': '\\\\(\\h{1,6}|.)'
-        'name': 'constant.character.escape.css'
+        'begin': "(?:\\G|^)(?=(?:[^\\\\']|\\\\.)+$)"
+        'end': '$'
+        'name': 'invalid.illegal.unclosed.string.css'
+        'patterns': [
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        'include': '#escapes'
+      }
+    ]
+  'tag-names':
```

Comment:
I am fine with whitelisting HTML tags to differentiate them from custom tags.
However, I am against whitelisting properties to differentiate them with invalid ones (https://github.com/atom/language-sass/issues/182#issuecomment-255596276).  I'm on the fence with whitelisting properties with no fallback highlighting - if the list of properties is very unlikely to be updated, then I think i'm fine with it.  However, if the list is constantly changing (eg property names), then I think it's best to not include a whitelist.